tests/examples/hello/src/iota.cc

```diff
@@ -0,0 +1,28 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class IotaTask : public Task<IotaTask, IOTA> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &output = context.outputs()[0];
+    auto output_shape = output.shape<1>();
+    auto out = output.write_accessor<float, 1>();
+    auto n = output_shape.volume();
+    for (size_t i = 0; i < n; ++i) {
+      out[i] = i + 1;
+    }
```

Comment:
Will this work on multiple processors/nodes? Each point task may receive a subset of the operation's "global" domain, and the Accessor class works with "global" indices, as opposed to local ones. I.e. I expect the code would need to look something like this.

```
for (int64_t i = output_shape.lo; i <= output_shape.hi; ++i) {
  out[i] = i + 1;
}
```

But maybe I'm missing something.

Reply:
It would be good to add some comments explaining this "local" vs "global" indexing.

Reply:
👍